src/components/layout/HostNav.tsx

```diff
@@ -0,0 +1,16 @@
+import { NavLink } from "react-router-dom";
+
+export default function HostNav() {
+    return (
+        <section className='mt-16 bg-[#FFF7ED] px-5 fixed w-full left-0 right-0 top-0 '>
+            <div className="max-w-6xl mx-auto py-6 flex items-center justify-between text-xl">
+                <div className="flex font-semibold items-center cursor-pointer gap-x-10">
+                    <NavLink to="/host" end className="hover:underline">Dashboard</NavLink>
```

Comment:
You can use a relative path rather than an absolute path for the `/host` path.
```suggestion
                    <NavLink to="." end className="hover:underline">Dashboard</NavLink>
```